src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -39,12 +55,21 @@
   private String deviceName;
   @Getter
   private Map<String, Object> deviceMetrics;
+  @Getter
+  private String userAgent;
 
   Emulator(String deviceName) {
     this.deviceName = deviceName;
+    this.userAgent = "";
   }
 
   Emulator(Map<String, Object> deviceMetrics) {
     this.deviceMetrics = deviceMetrics;
+    this.userAgent = "";
+  }
+
+  Emulator(Map<String, Object> deviceMetrics, String userAgent) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "Emulator" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)